juju/python-libjuju
# Cross model relations: Offers

Adding cross model offer add offer, remove offer and list offer.
The following also includes integration test around add and
removing offers.

## QA

You need to prepare a juju before doing the following:

```console
juju bootstrap lxd test --no-gui
```

Ensure that the example works by doing:

```console
tox -e example -- examples/crossmodel.py
```

Ensure that integration also works:

```console
tox -e integration -- tests/integration/test_crossmodel.py
```
